src/theme.ts

```diff
@@ -1,5 +1,5 @@
-import { createTheme } from '@mui/material/styles'
 import { red } from '@mui/material/colors'
+import { createTheme } from '@mui/material/styles'
```

Comment:
What's with the re-ordered imports in all of the files on this PR? 

There were no changes in this file other than the re-ordered imports, which means the changes in this file could be excluded from the PR.

Reply:
Yea I run prettier automatically so when I save files it re-orders these imports. It might just be vscode-I'll remove this see if I can't disable this.